specs/AppNotifications/AppNotifications-spec.md

```diff
@@ -451,6 +474,9 @@ namespace Microsoft.Windows.AppNotifications
         // For Unpackaged apps, the caller process will be registered as the COM server. And assets like displayname and icon will be gleaned from Shell and registered as well.
         void Register();
 
+         // For Unpackaged apps only, the caller process will be registered as the COM server.
```

Comment:
Wrong comment. The App is a COM server for both Packaged and Unpackaged apps. We call CoREgister for both. The only difference is that unpackaged apps can override their assets.

Reply:
The comment above this is also confusing